authtoken/authtoken.go

```diff
@@ -162,6 +178,8 @@ func InitTokenStore(config Configuration) Store {
 		store = NewFileStore(config.Path, config.Expiry)
 	case "redis":
 		store = NewRedisStore(config.Path, config.Prefix, config.Expiry)
+	case "stateless":
```

Comment:
`jwt`? Seem expose the implementation to developer will save use from explanation on how we do the stateless store.